tests/unit/dataactvalidator/test_scripts_loadFSRS.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+import pytest
+
+from dataactcore.models.fsrs import FSRSAward, FSRSSubaward
+from dataactvalidator.scripts import loadFSRS
+from tests.unit.dataactcore.factories import (
+    FSRSAwardFactory, FSRSSubawardFactory)
+
+
+@pytest.yield_fixture()
+def no_award_db(database):
```

Comment:
At the risk of outing myself for missing something obvious, I can't wrap my head around what `database` is and how it's working. I get that this stands in for a db in the unit tests, but was unable to find anything helpful in the pytest fixtures docs. Do you know of a good resource for grokking this?